src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -88,14 +88,8 @@ private static string ResolveExecutableFromPath(string executable, ref string ar
             if (useCmd)
             {
                 var comSpec = Environment.GetEnvironmentVariable("ComSpec");
-
-                // cmd doesn't like "foo.exe ", so we need to ensure that if
-                // args is empty, we just run "foo.exe"
-                if (!string.IsNullOrEmpty(args))
-                {
-                    executable = (executable + " " + args).Replace("\"", "\\\"");
-                }
-                args = $"/C \"{executable}\"";
+                // wrap 'executable' within quotes to deal woth space in its path.
```

Comment:
@livarcocc my understanding is that the outside quotes get stripped

```
2.  Otherwise, old behavior is to see if the first character is
        a quote character and if so, strip the leading character and
        remove the last quote character on the command line, preserving
        any text after the last quote character.
```